src/extensions/extensionInterface.ts

```diff
@@ -62,12 +62,9 @@ export interface MetricPlugin extends LabboxPlugin {
 
 // Sorting curation
 export type SortingCuration = {
-    labelsByUnit: {[key: string]: string[]}
-    labelChoices: string[]
-}
-export const defaultSortingCuration: SortingCuration = {
-    labelChoices: ["accept", "reject", "mua", "artifact", "noise"].sort(),
-    labelsByUnit: {}
+    labelsByUnit?: {[key: string]: string[]}
+    labelChoices?: string[]
+    mergeGroups?: (number[])[]
```

Comment:
This is the new mergeGroups field of the curation. An example: `[[1, 2, 4], [3, 9]]`

Reply:
The simplifyMergeGroups ensures that merge groups are disjoint. So when dispatch actions are applied, it should address this issue. Yes mergeGroupForUnit(unitId) should be a convenience function that we build into the interface. Making an issue for this.